waku/node/v2/config.nim

```diff
@@ -73,6 +73,21 @@ type
       defaultValue: ""
       name: "storenode" }: string
 
+    store* {.
+      desc: "Flag whether to start store protocol",
+      defaultValue: false
+      name: "store" }: bool
+
+    filter* {.
+      desc: "Flag whether to start filter protocol",
+      defaultValue: false
+      name: "filter" }: bool
+    
+    relay* {.
+      desc: "Flag whether to start relay protocol",
+      defaultValue: false
```

Comment:
I guess this can be true? Or one of the protocols at least? Else nothing starts default?

Reply:
This is still false.